README.md

```diff
@@ -1,2 +1,63 @@
 # get-aws-account-id-gh-action
-Retrieves aws account id
+Convenient way of retrieving aws account id
+
+### github-action
+
+This action retrieves aws account id based on account tag. The (required) inputs are:
+
+- _aws-account-tag_: AWS account tag used for retrieving id.
+- _aws-access-key_: AWS access key for accessing organizations.
+- _aws-secret-key_: AWS secret key for accessing organizations.
+- _aws-role-arn_: AWS role arn for accessing organizations.
+
+Outputs:
+
+- _aws-account-id_: AWS account id.
+
+### usage
+
+```yaml
+name: Deploy
+
+on:
+  workflow_call:
+    inputs:
+      stage:
+        description: 'Stage'
+        type: string
+        required: true
+        default: tst
+      client:
+        description: "Name of client"
+        type: string
+        required: true
+      account_prefix:
+        description: "Prefix of account to deploy to (ohp, dvc)"
+        type: string
+        required: false
+        default: 'dvc'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ohpensource/get-aws-account-id-gh-action@0.0.0.1
+        id: aws_account
+        name: Get account id
+        env:
```

Comment:
remove ENV variable and move definition to action field

Reply:
why?

Reply:
because it is standard format send all values to action via input parameters and not environment variables

Reply:
and now you have duplicity **AWS_ACCOUNT_TAG** and also **aws-account-tag** - and this is not necessary